engine/glfw/lib/android/android_init.c

```diff
@@ -59,6 +59,11 @@ static ASensorRef g_accelerometer = 0;
 static int g_accelerometerEnabled = 0;
 static uint32_t g_accelerometerFrequency = 1000000/60;
 static GLFWTouch* g_MouseEmulationTouch = 0;
+uint32_t g_EventLock = 0;
+int g_AppCommands[MAX_APP_COMMANDS];
+int g_NumAppCommands = 0;
+struct InputEvent g_AppInputEvents[MAX_APP_INPUT_EVENTS];
+int g_NumAppInputEvents = 0;
```

Comment:
These are used to pass events from the "looper_main" thread to "engine_main" thread in a thread safe manner.